### PR TITLE
Fix/warning adjustment

### PR DESCRIPTION
**Pull request by iamstarkov**
Currently, it is confusing to parse it because investigation pointers (rule, Component)
are in the same flow as warning's text itself. This subtle change aims
to put focus on important parts of the warning.

Before:
  Warning: [JSS] Could not find the referenced rule icon in Button, Themed, Static.

After
  Warning: [JSS] Could not find the referenced rule "icon" in "Button, Themed, Static".

### CODE REVIEW

**Comment by iamstarkov** [comment]
I dont know why my test is failing, thus I will appreciate guidance. rest of the test I assume are failing from the master

File: packages/jss-plugin-nested/src/index.test.js [hunk 1]
```diff
@@ -405,7 +405,7 @@ describe('jss-plugin-nested', () => {
       expect(spy.callCount).to.be(1)
       expect(
         spy.calledWithExactly(
-          'Warning: [JSS] Could not find the referenced rule "b" in ".a-id {\n  & $b: [object Object];\n}."'
+          'Warning: [JSS] Could not find the referenced rule "b" in ".a-id {\n  & $b: [object Object];\n}".'
```
Comment (iamstarkov) [inline packages/jss-plugin-nested/src/index.test.js]: I see

**Comment by kof** [comment]
merged,thx

**Comment by iamstarkov** [comment]
🙏

**Pull request merged**